Analytics/importers/scheduler_new.py

```diff
@@ -55,6 +56,43 @@ def status_has_changed(self, status: ImporterStatuses):
             importer_status.timestamp = datetime.now()
             importer_status.commit()
 
+            if status.state == "failure" and not importer_status.retrying:
+                importer_status.retrying = True
+                importer_status.save()
+                importer_status.commit()
+                try:
+                    Scheduler.retry_importer(status.name)
+                    logger.info("Exponential retry procedure was "
+                                "successful for class {}".format(status.name))
+                except Exception:
```

Comment:
General exception catching is not good practice. Change this to the expected exception(s) that are raised